mlflow/models/evaluation/default_evaluator.py

```diff
@@ -69,6 +71,32 @@
 _LATENCY_METRIC_NAME = "latency"
 
 
+@contextmanager
+def _suppress_class_imbalance_errors(exception_type=Exception, log_warning=True):
```

Comment:
Are we going to use `log_warning` param in some other place? It seems we are not using it in this PR.

Reply:
Thanks for the catch - I'm going to disable the logging for the redundant plot generation catches since the underlying data failure to calculate is the root cause of the failure to log the plot artifact.